microsoft-edge/devtools-guide-chromium/console/console-dom-interaction.md

```diff
@@ -5,290 +5,345 @@ author: MSEdgeTeam
 
-1. In the **Console**, hover over the resulting HTML `<header>` element, or press **Shift+Tab**.  In the rendered webpage, DevTools highlights the header:
+1. In the **Console**, hover over the resulting HTML `<main>` element, or press **Shift+Tab**.  In the rendered webpage, DevTools highlights the corresponding `<main>` element:
 
-   ![DevTools highlights the section you select in the Console](./console-dom-interaction-images/console-dom-highlight-element.png)
+   ![The main element, highlighted in the rendered webpage](./console-dom-interaction-images/highlight-on-hover.png)
 
 
 <!-- ====================================================================== -->
-## Manipulate the DOM
+## Change the DOM of the webpage
 
-You can manipulate the webpage from the **Console**, as follows.  In this example, you set a value in the DOM by using the Console, to affect the webpage styling: you add a green border around the header.
+You can make changes to the DOM tree of the webpage from the **Console** tool. In this example, you set the value of a DOM element attribute by using the Console, to affect the element's styling: you add a grey background to the `<h1>` title of the page.
 
-1. Press **Ctrl+Shift+J** (Windows, Linux) or **Command+Option+J** (macOS).  The Console opens in DevTools, next to the present webpage.
+1. Go to the webpage that you want to change. For example, open the [DevTools Console tool DOM interactions demo page](https://microsoftedge.github.io/Demos/devtools-console-dom-interactions/) in a new tab or window.
+
+1. Press **Ctrl+Shift+J** (Windows, Linux) or **Command+Option+J** (macOS).  The **Console** tool opens in DevTools, next to the webpage.
 
 1. Paste the following code into the **Console**:
 
    ```javascript
-   document.querySelector('header').style.border = '2em solid green'
+   document.querySelector("h1").style.backgroundColor = "lightgrey";
    ```
 
-   A green border appears around the header:
+   A grey background appears behind the page title:
 
-   ![Adding a border to an element by using the Console](./console-dom-interaction-images/console-dom-add-border.png)
+   ![The Console tool, showing the above JavaScript code, and the webpage showing the grey title](./console-dom-interaction-images/set-style.png)
 
 
-### Get a direct reference an element
+#### Get a reference to an element
 
-Depending on the complexity of the webpage, it can be daunting to find the right element to manipulate.  But you can use the **Inspect** tool to help you.  Suppose you want to manipulate the **Documentation** region within the header of the rendered page:
+Depending on the complexity of the webpage, it can be daunting to find the right element to change.  But you can use the **Inspect** tool to help you.  Suppose you want to change the text of the first link within the **Section 1** element of the rendered page:
```

Comment:
```suggestion
In a complex webpage, it can be hard to find the right element to change.  But you can use the **Inspect** tool to help you.  Suppose you want to change the text of the first link within the **Section 1** element of the rendered page:
```